config/initializers/withdrawal_reasons.rb

```diff
@@ -41,6 +41,7 @@ module WithdrawalReasons
   HAD_TO_WITHDRAW_TRAINEE_ANOTHER_REASON = "had_to_withdraw_trainee_another_reason"
   TRAINEE_CHOSE_TO_WITHDRAW_ANOTHER_REASON = "trainee_chose_to_withdraw_another_reason"
   MOVED_TO_DIFFERENT_ITT_COURSE = "moved_to_different_itt_course"
+  TRAINEE_WORKLOAD_ISSUES = "trainee_workload_issues"
```

Comment:
@kennyevil this was in the prototype, seemed to be missing so added here - I re-ran the migration that upsets these locally, worth doing the same your end!